e2e/support/commands/overwrites/log.js

```diff
@@ -1,14 +1,40 @@
-Cypress.Commands.overwrite("log", (originalFn, message) => {
-  Cypress.log({
-    displayName: `--- ${window.logCalls}. ${message} ---`,
-    name: `--- ${window.logCalls}. ${message} ---`,
+Cypress.Commands.overwrite("log", (originalFn, text) => {
+  const logConfig = {
+    displayName: `=== ${window.logCalls}. ${text}`,
+    name: "log",
     message: "",
-  });
+  };
 
+  appendStyleIfItDoesntExists(logConfig);
+  Cypress.log(logConfig);
   window.logCalls++;
 });
 
 // We want to reset the log counter for every new test (do not remove from this file)
 beforeEach(() => {
   window.logCalls = 1;
 });
+
+function appendStyleIfItDoesntExists(logConfig) {
+  const headHTML = Cypress.$(window.top.document.head);
+  const allStyles = Array.from(Cypress.$(window.top.document.styleSheets));
+
+  const getCustomStyle = allStyles.filter(s => s.ownerNode.dataset.customLog);
+  const customStyleExists = getCustomStyle.length > 0;
+
+  if (!customStyleExists) {
+    const style = document.createElement("style");
+
+    style.textContent = `
+      .command-name-${logConfig.name} .command-pin-target{
+        color: #ffffff !important;
+        background-color: darkorchid !important;;
```

Comment:
A bit more subtle variant:

```suggestion
        background-color: rgb(255, 255, 255, 0.2) !important;
```

![image](https://github.com/metabase/metabase/assets/6830683/5a0e98e1-0ee3-449a-a829-0b623d5a200e)

Reply:
Or we could reuse `background-color` of `.reporter-tag.reporter-tag-content.route`:

```suggestion
        background-color: #7f43c9 !important;
```

![image](https://github.com/metabase/metabase/assets/6830683/db2af07b-4280-4f8c-a696-500347be785e)

Reply:
Works for me :)